pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -156,6 +159,18 @@ router.post('/', (req, res, next) => {
                 res.redirect(req.originalUrl);
             });
         });
+    } else if (req.body.__action == 'regrade_question') {
+        if (!config.regradeActive) {
+            res.status(204).send();
+            return;
+        }
+        question.regradeQuestion(req.body.instance_question_id, err => {
```

Comment:
Rather than calling this directly, I think we should use `lib/server-jobs.js` to run in the background. That would handle the case where the grading is slow, and it would also let us output errors to the instructor.

An example of what this looks like is here: https://github.com/PrairieLearn/PrairieLearn/blob/42adbd08eb8afb5ab7f8122aed1fa5e58c12e1a1/pages/instructorAssessmentUploads/instructorAssessmentUploads.js#L29

Observe that we get a `job_sequence_id` and redirect to that page. The actual code starts a job sequence: https://github.com/PrairieLearn/PrairieLearn/blob/42adbd08eb8afb5ab7f8122aed1fa5e58c12e1a1/lib/score-upload.js#L49 and returns the `job_sequence_id` before starting the actual work: https://github.com/PrairieLearn/PrairieLearn/blob/42adbd08eb8afb5ab7f8122aed1fa5e58c12e1a1/lib/score-upload.js#L51

The code then continues and does the work. We can send information to the user by treating the `job` as a logger, like this: https://github.com/PrairieLearn/PrairieLearn/blob/42adbd08eb8afb5ab7f8122aed1fa5e58c12e1a1/lib/score-upload.js#L73

One other thing that we'll want to consider is that we shouldn't create issues and mark variants as broken while we are regrading. Instead we should print the errors in the job output, as above.